src/ray/object_manager/pull_manager.cc

```diff
@@ -236,23 +251,34 @@ void PullManager::UpdatePullsBasedOnAvailableMemory(size_t num_bytes_available)
 
   // While we are over capacity, deactivate requests starting from the back of the queues.
   DeactivateUntilWithinQuota("task args request", task_argument_bundles_,
+                             RayConfig::instance().pull_manager_min_active_pulls(),
```

Comment:
Don't we want this to be 0 if plasma_unlimited is off and 1 otherwise? What about removing the flag from the config and just setting it to 0 or 1 according to plasma_unlimited in this class?

Reply:
I'd like to keep the flag for debugging purposes (see the comment in ray_config_def.h)

Reply:
Hmm okay. Could we set it in the constructor though? I think it's a bit cleaner than referencing the config value everywhere.